flow/designs/nangate45/black_parrot/rules-base.json

```diff
@@ -56,19 +56,19 @@
         "compare": "<="
     },
     "finish__timing__setup__ws": {
-        "value": -2.84,
+        "value": -4.06,
```

Comment:
This doesn't look good.

Reply:
The clock is 
create_clock [get_ports clk_i]  -name CLK  -period 6.84  -waveform {0 3.42}

so it is a non-trivial shift.  @jjcherry56